Do not allow verifying email from a different account (#25711)
keycloak/keycloak: docs/documentation/release_notes/topics/24_0_0.adoc

In this release, the server will render the update profile page when the user is authenticating through a broker for the
first time using the `idp-review-user-profile.ftl` template.

For more details, see link:{upgradingguide_link}[{upgradingguide_name}].

= Performing actions on behalf of another user is not longer possible when the user is already authenticated

> In this release, it is no longer possible to perform actions, such as email verification, if the user is already authenticated

Reviewer: ```suggestion
In this release, you can no longer perform actions such as email verification if the user is already authenticated
```
